src/nagini_translation/resources/seq.sil

```diff
@@ -53,4 +53,43 @@ function Sequence___eq__(self: Ref, other: Ref): Bool
     requires issubtype(typeof(self), Sequence(Sequence_arg(typeof(self), 0)))
     requires issubtype(typeof(other), Sequence(Sequence_arg(typeof(other), 0)))
     requires Sequence_arg(typeof(self), 0) == Sequence_arg(typeof(other), 0)
-    ensures result == (Sequence___unbox__(self) == Sequence___unbox__(other))
\ No newline at end of file
+    ensures result == (Sequence___unbox__(self) == Sequence___unbox__(other))
+
+function PSet___create__(prim: Set[Ref], cont_type: PyType): Ref
```

Comment:
Again a bit of contradiction between sequence (ordered) and set (unordered). Should a set be in the seq.sil file?